### ISSUE TITLE
Local control

### ISSUE THREAD

**Issue by trommegutten** [kayjei/adax_wifi#10]
This method seems to work over the web and not directly offline?
Have you tried/had any luck flashing the adax chip with tasmota?

**Comment by kayjei** [comment]
No, I haven't tried it. It should also be able to communicate directly over LAN if one is able to solve the basic authentication used.

**Comment by parhedberg** [comment]
I have found out that in the app, while clicking on a heater the info about current temperature and firmware shows. If you then trippleclick the firmware version you get more info about the heater, as well as the password.. Could this be of any use in fixing a local control?

**Comment by kayjei** [comment]
I'm sure that could be useful. My guess is that the password is used for the encryption in some sense, unfortunately that is not the same as used as base64 basic auth.
My challenge is that I am unable to reverse engineer the way the encryption is generated (either the key if communicating with the cloud or for the basic authentication for local control), I need help from someone who is able to engineer the Adax app.

**Comment by marcelm** [comment]
In case it helps: Some pictures of the board can be found at stefanbode/Sonoff-Tasmota#157 .

**Comment by trommegutten** [comment]
Yes, maybe it is easier to just flash with Tasmota than reverse engineering the Adax app.
